keras_cv/models/segmentation/fcn.py

```diff
@@ -211,33 +168,29 @@ def vgg_architecture_builder(
 
     """
     if model_architecture == "fcn8s":
-        backbone = vgg_backbone_builder(
+        pool3, pool4, pool5 = vgg_backbone_builder(
             backbone=backbone,
             input_shape=input_shape,
             model_architecture="fcn8s",
             input_tensor=input_tensor,
         )
-        backbone_output = backbone(input_tensor)
-        pool3, pool4, pool5 = (
-            backbone_output["pool3"],
-            backbone_output["pool4"],
-            backbone_output["pool5"],
-        )
 
         pool3 = tf.keras.layers.Conv2D(
             filters=classes,
             kernel_size=(1, 1),
             padding="same",
             strides=(1, 1),
-            activation="relu",
+            activation="linear",
```

Comment:
Is "linear" an identity function? Why was relu replaced with it?

Reply:
It is, yes. I made the change as that is what was done in the other repository, and it seems to have yielded pretty good results indeed